src/Components/ClustersListTable/ClustersListTable.js

```diff
@@ -171,78 +175,88 @@ const ClustersListTable = ({
   };
 
   return (
-    <div id="clusters-list-table">
-      <PrimaryToolbar
-        pagination={{
-          itemCount: filteredRows.length,
-          page,
-          perPage: filters.limit,
-          onSetPage: (_event, page) =>
-            updateFilters({
-              ...filters,
-              offset: filters.limit * (page - 1),
-            }),
-          onPerPageSelect: (_event, perPage) =>
-            updateFilters({ ...filters, limit: perPage, offset: 0 }),
-          isCompact: true,
-          ouiaId: 'pager',
-        }}
-        filterConfig={{ items: filterConfigItems }}
-        activeFiltersConfig={activeFiltersConfig}
-      />
-      {(isUninitialized || isFetching) && <Loading />}
-      {isError && (
-        <Card ouiaId="error-state">
+    <>
+      {clusters.length === 0 ? (
+        <Card ouiaId="empty-state">
```

Comment:
Please, remove the card since there is no card in the mock-up.

Reply:
Fixed it. Updated the screenshot on top of PR